iocore/net/I_NetVConnection.h

```diff
@@ -482,6 +482,7 @@ class NetVConnection : public VConnection
 
   /** Returns remote sockaddr storage. */
   sockaddr const *get_remote_addr();
+  IpEndpoint const &get_remote_endpoint();
```

Comment:
Cannot make it const because internally these functions cache the remote address and set a bit indicating whether the value is already cache or not.